packages/vite-plugin-nitro/src/lib/vite-plugin-nitro.ts

```diff
@@ -77,6 +81,14 @@ export function nitro(options?: Options, nitroOptions?: NitroConfig): Plugin {
         ],
       };
 
+      if (buildPreset && buildPreset.toLowerCase().includes('vercel')) {
```

Comment:
We can also check for `process.env['VERCEL']` to detect Vercel in their CI environment so you don't have to set the `BUILD_PRESET`.

https://vercel.com/docs/concepts/projects/environment-variables/system-environment-variables

Reply:
added a check for it!